x/group/msgs.go

```diff
@@ -76,8 +70,7 @@ func (m Member) ValidateBasic() error {
 	if err != nil {
 		return sdkerrors.Wrap(err, "address")
 	}
-	if _, err := math.NewDecFromString(m.Weight); err != nil {
-		// if _, err := math.ParseNonNegativeDecimal(m.Weight); err != nil {
+	if _, err := math.NewPositiveDecFromString(m.Weight); err != nil {
```

Comment:
members weight can be 0 when updating, see previous comment
maybe we should actually use different logic for `MsgCreateGroup` and `MsgUpdateGroupMembers` because in the former, member weight can't be 0 while in the latter, member weight can be set to 0 to remove an existing member from the group